tests/semanticdb/expect/i9782.expect.scala

```diff
@@ -0,0 +1,20 @@
+// LazyRef
+trait Txn/*<-_empty_::Txn#*/[T/*<-_empty_::Txn#[T]*/ <: Txn[T]]
```

Comment:
We seems to be missing occurrences for `Txn[T]`

Reply:
Yeah... the part of `T <: Txn[T]` has the tree like:
```scala
TypeDef(T,TypeTree[TypeBounds(TypeRef(ThisType(TypeRef(NoPrefix,module class scala)),class Nothing),AppliedType(TypeRef(ThisType(TypeRef(NoPrefix,module class <empty>)),trait Txn),List(LazyRef(TypeRef(ThisType(TypeRef(ThisType(TypeRef(NoPrefix,module class <empty>)),trait Txn)),type T)))))])
```
and not sure how can we traverse into the `Txn[T]` part and get those symbols 🤔

Reply:
It's not important for this PR, but maybe we could raise a follow up issue?

Reply:
Also I don't think you would be able to extract any sensible spans from within a `TypeTree` without scanning source code - only trees have spans and the argument of `TypeTree` is a type.